stable_baselines3/dqn/dqn.py

```diff
@@ -170,6 +173,8 @@ def _on_step(self) -> None:
         self._n_calls += 1
         if self._n_calls % self.target_update_interval == 0:
             polyak_update(self.q_net.parameters(), self.q_net_target.parameters(), self.tau)
+            # Copy running stats, see GH issue #996
```

Comment:
I will leave it like that for now, but if in experiments it doesn't work, we should copy the stats after each gradient step